### PR TITLE
:sparkles: Add manual steps to connect EKS clusters

### PR DESCRIPTION
**Pull request by suvaanshkumar**
## Summary
Adding manual steps to connect EKS clusters as Hub and spoke using OCM
## Related issue(s)

Fixes #

### CODE REVIEW

File: solutions/joining-hub-and-spoke-with-aws-auth-manually/README.md [hunk 1]
```diff
@@ -0,0 +1,182 @@
+# Manually join Hub and Spoke using AWS-based authentication
+
+This guide provides list of manual steps on how we can manually join AWS EKS clusters as Hub and Spoke using OCM.
+
+# Background
+
+We are working on a new feature to run OCM natively on EKS with AWS IAM based authentication. More details, on why and how, can be found in [enhancement proposal](https://github.com/open-cluster-management-io/enhancements/blob/main/enhancements/sig-architecture/105-aws-iam-registration/README.md). This purpose of this document is to:
```
Comment (mikeshng) [inline solutions/joining-hub-and-spoke-with-aws-auth-manually/README.md:7]:
```suggestion
We are working on a new feature to run OCM natively on EKS with AWS IAM based authentication. More details, on why and how, can be found in [enhancement proposal](https://github.com/open-cluster-management-io/enhancements/blob/main/enhancements/sig-architecture/105-aws-iam-registration/README.md). The purpose of this document is to:
```

File: solutions/joining-hub-and-spoke-with-aws-auth-manually/README.md [hunk 2]
```diff
@@ -0,0 +1,182 @@
...
+- Share with community, 
+- sthe proof of design proposed in [enhancement proposal](https://github.com/open-cluster-management-io/enhancements/blob/main/enhancements/sig-architecture/105-aws-iam-registration/README.md)
```
Comment (mikeshng) [inline solutions/joining-hub-and-spoke-with-aws-auth-manually/README.md:9]:
```suggestion
- Show the proof of design proposed in [enhancement proposal](https://github.com/open-cluster-management-io/enhancements/blob/main/enhancements/sig-architecture/105-aws-iam-registration/README.md)
```

File: solutions/joining-hub-and-spoke-with-aws-auth-manually/README.md [hunk 3]
```diff
@@ -0,0 +1,182 @@
...
+    export HUB_ROLE_NAME=ocm-hub-138c525cf3b7f74ba3b945e7847792cc
+    export SPOKE_ROLE_NAME=ocm-managed-cluster-138c525cf3b7f74ba3b945e7847792cc
+    export HUB_POLICY_NAME=hub
+    export SPOKE_POLICY_NAME=spoke
+    export SPOKE_OIDC_PROVIDER_ID=BE6D831CFA22823093D17E608EE0048C
+    export HUB_REGION=us-west-2
+    export SPOKE_REGION=us-west-2
+   ```
+
+2. Login to spoke AWS account and create IAM role, policy and tags on spoke IAM. Go to the folder containing this markdown file in this [repository](https://github.com/open-cluster-management-io/open-cluster-management-io.github.io) and run following commands:
```
Comment (mikeshng) [inline solutions/joining-hub-and-spoke-with-aws-auth-manually/README.md:50]:
```suggestion
2. Login to spoke AWS account and create IAM role, policy and tags on spoke IAM. Go to the folder containing this markdown file in this [repository](https://github.com/open-cluster-management-io/ocm/tree/main/solutions/joining-hub-and-spoke-with-aws-auth-manually) and run following commands:
```

**Review by mikeshng — COMMENTED** [review]
Looks great! Thanks for this guide. 

I will give it a try next week.

File: solutions/joining-hub-and-spoke-with-aws-auth-manually/README.md (hunk 3 above)
Comment (jaswalkiranavtar) [reply solutions/joining-hub-and-spoke-with-aws-auth-manually/README.md]: fixed.

File: solutions/joining-hub-and-spoke-with-aws-auth-manually/README.md (hunk 2 above)
Comment (jaswalkiranavtar) [reply solutions/joining-hub-and-spoke-with-aws-auth-manually/README.md]: fixed.

**Review by mikeshng — APPROVED** [review]
/approve

/lgtm

Thanks!

**Pull request merged**